src/ontology/imports/other_import.owl

```diff
@@ -51,6 +54,25 @@ SubClassOf(<http://purl.obolibrary.org/obo/OGMS_0000031> <http://purl.obolibrary
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/UPHENO_0001001> "phenotype"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/UPHENO_0001001> <http://purl.obolibrary.org/obo/PATO_0000001>)
 
+# Class: <http://purl.obolibrary.org/obo/IAO_0000109> (measurement datum)
+
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/IAO_0000109> "measurement datum")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/IAO_0000109> "A measurement datum is an information content entity that is a recording of the output of a measurement such as produced by a device."@en)
+SubClassOf(<http://purl.obolibrary.org/obo/IAO_0000109> <http://purl.obolibrary.org/obo/IAO_0000027>)
```

Comment:
Please, import the COB term [COB:0000121](https://www.ebi.ac.uk/ols4/ontologies/cob/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCOB_0000121?lang=en) instead and remove this entry here.